src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,100 @@
+package core.basesyntax;
+
+import core.basesyntax.exception.RegistrationDataException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static User user;
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @BeforeAll
+    static void createValidUser() {
+        user = new User();
+        user.setLogin("validLogin");
+        user.setPassword("validPassword");
+        user.setAge(33);
+    }
```

Comment:
The user instance should be created fresh for each test to ensure test independence. Instead of using a static user, consider initializing a new User object in each test or in a method annotated with @BeforeEach.